mmv1/products/metastore/Service.yaml

```diff
@@ -515,3 +515,12 @@ properties:
         values:
           - :LEGACY
           - :JSON
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: 'tags'
+    description: |
+      A map of resource manager tags.
+      Resource manager tag keys and values have the same definition as resource manager tags.
+      Keys must be in the format tagKeys/{tag_key_id}, and values are in the format tagValues/456.
```

Comment:
nit: can be tagValues/{tag_value_id} instead of tagValues/456
here and other instances in the PR.

Reply:
Done